tests/test_imod_coupler/test_ribamod.py

```diff
@@ -135,7 +135,7 @@ def test_ribamod_backwater(
         run_coupler_function=run_coupler_function,
     )
 
-    final_level = results.basin_df[results.basin_df["time"] == "2021-01-01"]["level"]
+    final_level = results.basin_df[results.basin_df["time"] == "2020-12-31"]["level"]
```

Comment:
Why do you change this? You extend the model period, but take the water level after one year?

Reply:
There is a problem with the first day of the new year, which seems just beyond the range of acceptable entries, that is why the final_level was inquired a day earlier, just at the end of the simulation. The extension with two years was only for the bucket model (because the bucket seems to empty slower now) and not for the backwater model.